src/com/goide/GoDocumentationProvider.java

```diff
@@ -102,6 +104,25 @@ else if (type == GoParserDefinition.MULTILINE_COMMENT) {
         }
         return text;
       }
-    }), "<br/>") + "</pre>";
+    }), "<br/>");
+  }
+
+  private static String getSignature(PsiElement element) {
+    if (element instanceof GoFunctionDeclaration) {
+      return "func " +
+             ((GoFunctionDeclaration)element).getIdentifier().getText() +
+             ((GoFunctionDeclaration)element).getSignature().getText() +
+             "<br/><br/>";
```

Comment:
I was looking at the screenshot in the comment here: https://github.com/go-lang-plugin-org/go-lang-idea-plugin/issues/1234#issuecomment-72492954 and trying to give it a distinction in relation to the documentation.
Should I rather bold the the function signature?